test/util/artifacts/runas-memleak/README.md

```diff
@@ -48,42 +48,78 @@ instance of ExecuteShellCommand
 }
 
 Starting values for omiagent process:
-PID 33704
+PID 29574
 Thread count 3
-FD count 42
+FD count 40
 Memstats:
   PID   RSS    VSZ COMMAND
-33704  5740 302932 omiagent
-```
+29574  4608  16348 omiagent
 
-##### Output from Stabilization
+Will now exercise Provide_ExShell_Load RunAs provider under load:
+..............
 
-```
-Will now exercise RunAs provider under load:
-........................................
+Intermediate values for Provide_ExShell_Load RunAs provider:
+PID 29574
+Thread count 3
+FD count 40
+Memstats:
+  PID   RSS    VSZ COMMAND
+29574  4608  16348 omiagent
+
+Will now exercise Provide_ExScript_Load RunAs provider under load:
+..............
 
-Intermediate values for RunAs provider:
-PID 33704
+Intermediate values for Provide_ExScript_Load RunAs provider:
+PID 29574
 Thread count 3
-FD count 42
+FD count 40
 Memstats:
   PID   RSS    VSZ COMMAND
-33704  5880 368468 omiagent
-```
+29574  4608  16348 omiagent
 
-##### Output from Leak Detection
```

Comment:
This, too, appears to be missing. This needs to be restored where it belongs now.

Or, if you can't restore it, then rework the document to not mention them at all. But I thought it helped to have them, to explain what was going on. Look at original document for clarity on what I mean.